main.tf

```diff
@@ -53,13 +49,15 @@ module "kubeflow-cluster" {
 
 # Deploy KFP CRDs
 module "kfp-apply-crds" {
-  source                = "terraform-google-modules/gcloud/google"
-  module_depends_on     = [module.kubeflow-cluster.endpoint, local_file.params.content, local_file.params-db.content]
-  platform              = "linux"
-  additional_components = ["kubectl", "beta"]
+  source = "github.com/terraform-google-modules/terraform-google-gcloud//modules/kubectl-wrapper"
 
-  create_cmd_entrypoint = "${path.module}/scripts/kubectl_wrapper.sh"
-  create_cmd_body       = "https://${module.kubeflow-cluster.endpoint} ${data.google_client_config.default.access_token} ${module.kubeflow-cluster.ca_certificate} kubectl apply -k ${local.crd_path}"
+  module_depends_on = [module.kubeflow-cluster.endpoint, local_file.params.content, local_file.params-db.content]
+  cluster_name      = module.kubeflow-cluster.name
+  cluster_location  = module.kubeflow-cluster.location
+  project_id        = var.project_id
+
+  kubectl_create_command  = "kubectl apply -k ${local.crd_path}"
+  kubectl_destroy_command = "kubectl delete -k ${local.crd_path}"
```

Comment:
Do we still need the destroy command if we're destroying the entire cluster?

Reply:
yeah I think it makes sense if they want to do an upgrade in place, i.e delete and reapply new CRD